src/ASM/LR/ASMu2D.C

```diff
@@ -1754,6 +1758,12 @@ bool ASMu2D::integrate (Integrand& integrand,
   int iel = 0;
   for (const LR::Element* elm : lrspline->getAllElements())
   {
+    if (!myElms.empty() &&
+        std::find(myElms.begin(), myElms.end(), iel) == myElms.end()) {
+        ++iel;
```

Comment:
For consistency (see similar change for ASMu2Dmx below), increment iel before the if-statement instead

Reply:
ok.